src/Components/Rides/RiderSignup/RiderSignup.tsx

```diff
@@ -2,36 +2,71 @@
  * Rider signup form.
  */
 
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Button, Col, Container, Form, Row } from 'react-bootstrap';
 import { Turnarounds } from '../../../Constants/RidesConstants';
+import { useNavigate } from 'react-router-dom';
 
 import './RiderSignup.scss';
-import 'css/common/forms.scss'
+import 'css/common/forms.scss';
 
 export const RiderSignup = () => {
-  const [disableApartmentTurnaroundTextInput, setDisableApartmentTurnaroundTextInput] = useState(true);
+  const navigate = useNavigate();
+
+  const [
+    disableApartmentTurnaroundTextInput,
+    setDisableApartmentTurnaroundTextInput,
+  ] = useState(true);
+  const [formSubmitted, setFormSubmitted] = useState(false);
+
+  const handleSubmit = () => {
+    // if validated then submit form
+    setFormSubmitted(true);
+  };
+
+  useEffect(() => {
+    let timer = setTimeout(() => {
+      if (formSubmitted) {
+        navigate('/');
+      }
+    }, 4000);
+    return () => clearTimeout(timer);
+  }, [formSubmitted, navigate]);
+
+  if (formSubmitted) {
+    // insert thank you page
+    return <div>Thank You!</div>;
+  }
 
   return (
     <Container>
       <Col className={'mx-auto text-center'} lg={'8'}>
         <span className={'signup-form-title'}>Sign up for a ride to Church!</span>
 
-        <Form className={'text-center'}>
+        <Form className={'text-center'} onSubmit={handleSubmit}>
           <Row className={'text-start gx-5'}>
             <Col lg={'6'}>
               <Form.Group className={'mb-3'} controlId={'riderName'}>
-                <Form.Label className={'signup-form-label'}> Name * </Form.Label>
+                <Form.Label className={'signup-form-label'}>
+                  {' '}
+                  Name *{' '}
+                </Form.Label>
                 <Form.Control placeholder={'Enter your name'} />
               </Form.Group>
 
               <Form.Group className={'mb-3'} controlId={'riderEmail'}>
-                <Form.Label className={'signup-form-label'}> Email * </Form.Label>
+                <Form.Label className={'signup-form-label'}>
+                  {' '}

```

Comment:
Why is this stuff added? Does it look weird on your computer?

Reply:
my formatter automatically added that i'll have to look into the settings of my formatter to change it